ui/pages/Dashboard/components/table-body/EditProjectCategoriesModal.jsx

```diff
@@ -13,16 +13,17 @@ const EditProjectCategoriesModal = (props) => {
   const categories = Object.values(props.projectCategoriesByGuid).map((projectCategory) => {
     return { value: projectCategory.guid, text: projectCategory.name, key: projectCategory.guid }
   })
+  const NAME = `editProjectCategories-${props.project.projectGuid}`
```

Comment:
I'm not sure what you mean

Reply:
sorry I meant either rename this const to camelCase (eg. "modalName"), or just use  the value (`editProjectCategories-${props.project.projectGuid}`) directly instead of creating a const first.

Reply:
oh got it. I fixed that